keps/sig-node/compute-device-assignment.md

```diff
@@ -38,7 +40,7 @@ status: implementable
 <!-- /toc -->
 
 ## Abstract
-In this document we will discuss the motivation and code changes required for introducing a kubelet endpoint to expose device to container bindings.
+In this document we will discuss the motivation and code changes required for introducing a kubelet endpoint to expose device and cpu ids to container bindings.
```

Comment:
rather than sparsely enumerate, is it better to say:

"Motivation and code changes required for introducing a kubelet endpoint to expose concrete resource assignments made to pods"  -- this keeps it loose enough that cpu, memory, numa, devices all fall under that bucket.